modules/sharing-registry/sharing-data-migrator/pom.xml

```diff
@@ -59,6 +59,16 @@
             <artifactId>airavata-data-models</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.airavata</groupId>
+            <artifactId>airavata-credential-store-stubs</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
```

Comment:
Do you need the services-security dependency?

Reply:
You're right. This is not needed. I added this earlier because I was using the KeyCloakSecurityManager instance in my code.